index.html

```diff
@@ -465,7 +474,13 @@ <h2>Processing Model</h2>
             </dl>
           </li>
           <li>
-            <a>Fetch</a> <var>req</var>.
+            Let <var>handleBeaconDone</var> given <a>response</a> <var>res</var> be to
```

Comment:
s/be to/be a call to/ ?
I found it initially hard to read, so the above may help. Alternatively, we can define this as a helper algorithm and call on it.